src/tests/Microsoft.PowerFx.Interpreter.Tests/CollectFunctionTests.cs

```diff
@@ -30,16 +30,17 @@ public class CollectFunctionTests : PowerFxTest
         [Theory]
         [InlineData("Collect(t, r1)", 1)]
         [InlineData("Collect(t, r1);Collect(t, r1);Collect(t, r1)", 3)]
-        [InlineData("Collect(t, r1);Collect(t, Blank())", 1)]
         [InlineData("Collect(t, r1);Collect(t, {})", 2)]
+
+        //[InlineData("Collect(t, r1);Collect(t, Blank())", 1)]
```

Comment:
>  //[InlineData("Collect(t, r1);Collect(t, Blank())", 1)] [](http://example.com/codeflow?start=6&length=57)

?

Reply:
This has been placed back. Thanks for catching it.